examples/app_commands/basic.py

```diff
@@ -57,16 +58,16 @@ async def add(interaction: discord.Interaction, first_value: int, second_value:
 
 
 # To make an argument optional, you can either give it a supported default argument
-# or you can mark it as Optional from the typing library. This example does both.
+# or you can mark it as Optional from the typing standard library. This example does both.
 @client.tree.command()
-@app_commands.describe(member='The member you want to get the joined date from, defaults to the user who uses the command')
+@app_commands.describe(member='The member you want to get the joined date from; defaults to the user who uses the command')
 async def joined(interaction: discord.Interaction, member: Optional[discord.Member] = None):
     """Says when a member joined."""
     # If no member is explicitly provided then we use the command user here
     member = member or interaction.user
 
     # The format_dt function formats the date time into a human readable representation in the official client
-    await interaction.response.send_message(f'{member} joined in {discord.utils.format_dt(member.joined_at)}')
+    await interaction.response.send_message(f'{member} joined on {discord.utils.format_dt(member.joined_at)}')
```

Comment:
Actually, now that I think about it, should it be joined "at" or joined "on?" "At" is used for precise time, for example, "at 5:32PM." "On" is used for general days, for example, "on April 30." However this is a combination of datetime, for example, "April 30, 2022 8:22AM." Either it would be best to omit the "at" or "on" keyword completely, or perhaps follow the library's choice of "joined_at."

Reply:
Ealier this did not use format_dt (as seen in the basic_bot.py example) and used `in`, I just copied it over from the basic bot example. But since english is not my native language, I cannot decide here.

Reply:
I think I will use "at" just to be consistent with the library's `joined_at` function. Honestly, it's an example so the significance isn't the highest.

Reply:
Seems fine.